cmd/validatorcmd/list.go

```diff
@@ -50,6 +53,14 @@ func list(_ *cobra.Command, args []string) error {
 	}
 
 	blockchainName := args[0]
+	sc, err := app.LoadSidecar(blockchainName)
```

Comment:
we should move this code up to the top of the func. including `blockchainName := args[0]`.
these is no point of doing networkoptions.GetNetworkFromCmdLineFlags if sidecar reports blockchain as not SoV